src/cloud/components/molecules/Editor/index.tsx

```diff
@@ -596,6 +597,12 @@ const Editor = ({ doc, team, user, contributors, backLinks }: EditorProps) => {
   )
 
   const { settings } = useSettings()
+  const fontSize = useMemo(() => {
```

Comment:
Don't overuse memo hooks for simple getters for primitive types, string, number and boolean.